composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "yotpo/module-review",
     "description": "Yotpo Reviews extension for Magento2",
-    "version": "3.1.0",
+    "version": "3.1.1",
```

Comment:
you need also to change the version in:
https://github.com/YotpoLtd/magento2-module-yotpo-reviews/blob/7c8218d35afad7bb79f4f7428ac979f9d82670d2/etc/module.xml

Then go to yotpo-ext, change the versions in the same places, and open PR for that.

NOTE: in this case do not cherry-pick the commit because the composer.json files are not the same.